packages/loader/container-definitions/src/browserPackage.ts

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IFluidPackageEnvironment, IFluidPackage, isFluidPackage } from "./fluidPackage";
+
+/**
+ * A specific Fluid package environment for browsers
+ */
+export interface IFluidBrowserPackageEnvironment extends IFluidPackageEnvironment{
+    /**
+     * The umd target specifics the scripts necessary for loading a packages
+     * in a browser environment
+     */
+    umd: {
+        /**
+         * The bundled js files for loading this package
+         */
+        files: string[];
+
+        /**
+         * The global name that the script entry points will be exposed.
+         * This entry point should be an IFluidModule
+         */
+        library: string;
+    },
+    /**
+     * The css target optionally specifics the necessary stylesheets for this package
+     */
+    css?: {
```

Comment:
I wonder if it needs to be more generic than this? Fonts, images, etc stand out to me as things we may want to download in parallel to container code.

Any use cases of this already? Office fluid container doesn't need this yet, but I'm curious if other containers are already doing this. One reason I bring this up is because some additional context (such as the ui language) may be needed as a hint from those host to determine the correct assets to download

Reply:
yeah. i don't have a great solution here. and i'm worried about complexity. open to ideas

Reply:
part of me thinks we should just allow scripts, and then the scripts should load anything else they need, but understand there are perf issues with that. a middle group might be to only support scripts, and pre-caching. thoughts?

Reply:
so i went ahead and added the umd for code, and targets for all supported browser cache targets. curious what you think

Reply:
I like your idea of only supporting scripts for now (but leaving room for the API to be modified later). That way we can come at this problem when we are motivated to support it with a real use case.